src/chatbot/src/chatbot/aiml/DefaultSubs.py

```diff
@@ -48,8 +50,8 @@
     "hers": "his",
     "herself": "himself",
 }
-
-defaultPerson = {
+# These now match the comment above
+defaultPerson2 = {

```

Comment:
Yikes!
https://www.chatbots.org/ai_zone/viewthread/1839/

seems to be saying that the definition of person and person2 was flipped between AIML version 1.0 and 2.0 and there is assorted confusion as a result.